assets/data/aspects/drauvenAging.json

```diff
@@ -5,114 +5,135 @@
 	"id": "drauven_cannotAge",
 	"lifespan": "all"
 },
-{
+
+{ 
 	"modId": "wildermyth-drauven-pcs",
-	"id": "drauven_earlyMiddleAge",
+	"id": "drauven_aging_armour",
```

Comment:
🤔 Have these new aspects been added to `aspects.properties` so that something nice shows up in the stat bonuses/penalties?

... Nope, they have not. Excellent catch.

Reply:
Updated!